.changeset/thirty-mayflies-travel.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react': major
```

Comment:
I don't think this should be a major change yet. We'll kind of block ourselves doing that. Once we make all the changes, we can graduate from drafts to main folder with a major change. Could you do that before merging?